standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/ObjectStore.java

```diff
@@ -11322,6 +11322,16 @@ public NotificationEventResponse getNextNotification(NotificationEventRequest rq
         filterBuilder.setLength(filterBuilder.length() - 4); // remove the last " || "
         filterBuilder.append(") ");
       }
+      if (rqst.isSetEventTypeList()) {
+        filterBuilder.append(" && (");
+        for (String eventType : rqst.getEventTypeList()) {
```

Comment:
nit: leave it to you, can we use `rqst.contains(eventType)` to the `filterBuilder`?